src/input/mod.rs

```diff
@@ -1817,14 +1817,18 @@ impl State {
             }
         }
 
-        let horizontal_amount = horizontal_amount.unwrap_or_else(|| {
-            // Winit backend, discrete scrolling.
-            horizontal_amount_v120.unwrap_or(0.0) / 120. * 15.
-        });
-        let vertical_amount = vertical_amount.unwrap_or_else(|| {
-            // Winit backend, discrete scrolling.
-            vertical_amount_v120.unwrap_or(0.0) / 120. * 15.
-        });
+        let scroll_factor = self.niri.config.borrow().input.touchpad.scroll_factor;
+
+        let horizontal_amount = scroll_factor
```

Comment:
Two problems doing the multiplication here:
1. It affects both mouse wheel and touchpad when the setting is in the `touchpad {}` section. I'd expect `mouse {}` to have the same, separate scroll factor setting, and them to apply only to the correct input device (the `source` check like above).
2. It doesn't affect the scroll bindings above. I don't think it should for the mouse wheel but maybe it should for touchpad? Not sure.